submissions/hadizaSaadu/HTML/E-commerce.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>product-page</title>
+  </head>
+  <body bgcolor="#e6ac00">
+    <h1><center>DEEZAH'S CLOSET</center></h1>
+    <h3>
+      <i><center>Home of modest wears . . .</center></i>
+    </h3>
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Is there a reason for `&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;` all other the place?

Reply:
> Is there a reason for `&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;` all other the place?

yes, i used it to space the elements.

Reply:
@bint-saad use <br> instead of &nbsp;.